apps/prairielearn/src/components/PageLayout.html.ts

```diff
@@ -0,0 +1,78 @@
+import { html, type HtmlValue } from '@prairielearn/html';
+
+import { HeadContents } from './HeadContents.html.js';
+import { Navbar } from './Navbar.html.js';
+import type { NavContext } from './Navbar.types.js';
+
+export function PageLayout({
+  resLocals,
+  pageTitle,
+  navContext,
+  options = {
+    marginBottom: true,
+  },
```

Comment:
Nit, this default is redundant.

```suggestion
  options = {},
```